docs/chat/chat-schema.yaml

```diff
@@ -46,3 +57,14 @@ definitions:
       updatedAt:
         type: string
         format: date-time
+      deletedFor:
+        type: array
+        items:
+          type: object
+          properties:
+            user:
+              type: object
```

Comment:
please, check the chat service, probably there are only ids, so it would be a string

Reply:
It's object in the service